src/mumble/AudioWizard.cpp

```diff
@@ -728,3 +737,16 @@ void AudioWizard::on_qrbQualityCustom_clicked() {
 	g.s.iFramesPerPacket = sOldSettings.iFramesPerPacket;
 	restartAudio();
 }
+
+/// Which echo cancellation is usable depends on the audio backend and the device combination.
+/// This function will iterate through the list of available echo cancellation in the audio backend and check with
+/// the backend whether this echo cancellation option works for current device combination.
+EchoCancelOptionID AudioWizard::firstUsableEchoCancellation(AudioInputRegistrar *air, const QString outputSys) {
```

Comment:
The docs should be at the declaration rather than the implementation :point_up: 
```suggestion
EchoCancelOptionID AudioWizard::firstUsableEchoCancellation(AudioInputRegistrar *air, const QString outputSys) {
```